webapp/certification/views.py

```diff
@@ -12,4 +12,69 @@
 
 
 def certification_home():
-    return flask.render_template("certification/index.html")
+
+    certified_releases = api.certified_releases(limit="0")["objects"]
+    certified_makes = api.certified_makes(limit="0")["objects"]
+
+    # Desktop section
+    desktop_releases = []
+    desktop_vendors = []
+
+    for release in certified_releases:
```

Comment:
If you declare all the list at the tops you can do all the appends in these two for loops

Reply:
Thanks that's a good one